forge-gui/res/cardsfolder/upcoming/scholar_of_new_horizons.txt

```diff
@@ -0,0 +1,13 @@
+Name:Scholar of New Horizons
+ManaCost:1 W
+Types:Creature Human Scout
+PT:1/1
+K:etbCounter:P1P1:1
+A:AB$ ChangeZone | Cost$ T RemoveAnyCounter<1/Any/Permanent.YouCtrl/a permanent you control> | Origin$ Library | Destination$ Library | ChangeType$ Land.Plains | ChangeNum$ 1 | RememberChanged$ True | Reveal$ True | Shuffle$ False | SubAbility$ DBBranch | SpellDescription$ Search your library for a Plains card and reveal it. If an opponent controls more lands than you, you may put that card onto the battlefield tapped. If you don't put the card onto the battlefield, put it into your hand. Then shuffle.
+SVar:DBBranch:DB$ Branch | BranchConditionSVar$ X | TrueSubAbility$ ToBattlefieldTapped | FalseSubAbility$ ToHand | SubAbility$ DBCleanup
```

Comment:
You should still be able to choose to have it go to your hand

Reply:
Line 8 still need some changes, thoughts ?

Reply:
What didn't work?
IsPresent$ is for restrictions

Reply:
Removing the cleanup seems to make it work. That ok rules wise ?

Reply:
The cleanup should have nothing to do with that
On permanents it should always be done to avoid conflicts with other abilities

Reply:
You're right. I think I finally got it working